gen3_util/files/middleware.py

```diff
@@ -8,7 +8,9 @@
 from gen3_util.files.lister import ls
```

Comment:
Should this be in its own file `middleware.py`? Only use is from files.cli consider moving to file.cli
@matthewpeterkort

Reply:
Was getting a circular import error when it was in it's original file.